x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/SqlQueryResponse.java

```diff
@@ -123,36 +78,6 @@ public boolean isRunning() {
         return isRunning;
     }
 
-    public static SqlQueryResponse fromXContent(XContentParser parser) {
-        return PARSER.apply(parser, null);
-    }
-
-    public static List<List<Object>> parseRows(XContentParser parser) throws IOException {
```

Comment:
These are not needed anymore - they just do parsing of a list of lists which is already supported.